### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by axenso** [avocode/avocode#1804]
---
* **Version**: 2.17.1
* **Sketch plugin version**: 47.1.4
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 17107
---
```
TypeError: Cannot read property 'get' of undefined
    at e.getLayerIdsToSelect (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2675213)
    at e.mouseup (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2672952)
    at e.mouseup (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2663645)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2667214
    at wrapped (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:314:29)
```

**Comment by vojtatranta** [comment]
hi @axenso ,
this will be fixed in the next release (tommorow)

**Comment by bolekkerous** [comment]
Hey - 
we’ve just released a new version - 2.19 - which I believe should fix this issue. 

Please update your app and it should hopefully work. 

You can check our changelog for the full release notes:

https://avocode.com/changelog

If you still encounter this or any other issue, please reopen or create a new issue.

Best regards,
Bolek, Avocode Team
